### PR TITLE
Improve bazel rbe cache hit rate

### PR DESCRIPTION
**Pull request by fejta**
/assign @BenTheElder 

Not entirely sure what is up with these two targets, so just run them locally for now.

### CODE REVIEW

File: prow/interrupts/BUILD.bazel [hunk 1]
```diff
@@ -26,4 +26,5 @@ go_test(
     name = "go_default_test",
     srcs = ["interrupts_test.go"],
     embed = [":go_default_library"],
+    tags = ["local"],  # TODO(fejta): non-hermetic tests fails on RBE
```
Comment (BenTheElder) [inline prow/interrupts/BUILD.bazel]: is there an issue for this?

**Comment by BenTheElder** [comment]
/lgtm
seems like the tests only being run locally should have a tracking issue to ensure they don't stay that way?

**Comment by fejta** [comment]
Maybe they are just flaky? I'm going to see if I can turn it off

**Pull request merged**